nautobot/utilities/filters.py

```diff
@@ -492,20 +487,23 @@ class TreeNodeMultipleChoiceFilter(NaturalKeyOrPKMultipleChoiceFilter):
     would match both "Athens" and "Durham".
     """
 
-    def __init__(self, *args, **kwargs):
-        kwargs.setdefault("lookup_expr", "in")
-        super().__init__(*args, **kwargs)
-
     def filter(self, qs, value):
+        new_value = []
         if value:
-            if any(isinstance(node, TreeNode) for node in value):
-                # django-tree-queries
-                value = [node.descendants(include_self=True) if not isinstance(node, str) else node for node in value]
-            elif any(isinstance(node, MPTTModel) for node in value):
-                # django-mptt
-                value = [
-                    node.get_descendants(include_self=True) if not isinstance(node, str) else node for node in value
-                ]
+            for node in value:
+                if isinstance(node, TreeNode):
+                    method = "descendants"
+                elif isinstance(node, MPTTModel):
+                    method = "get_descendants"
+                else:
+                    continue
```

Comment:
Should we at least log an error in here or something?

Reply:
I'm realizing I didn't refactor this correctly. It's not meant to be an error case.